app/code/core/Mage/AdminNotification/Model/Feed.php

```diff
@@ -169,7 +169,7 @@ public function getFeedData()
         $curl->close();
 
         try {
-            $xml  = new SimpleXMLElement($data);
+            $xml = @new SimpleXMLElement($data);
```

Comment:
what happens now if an exception is thrown?

Reply:
Wow! I'm stupid, ignore that.